signing/signing.go

```diff
@@ -0,0 +1,65 @@
+package signing
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	JsonOutput string
+}
+
+var errorPullingBucket = errors.New("error pulling from GCS bucket")
+var errorVerifyingFilepath = errors.New("error verifying filepath format")
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	host := mux.Vars(r)["host"]
+	orgName := mux.Vars(r)["orgName"]
+	repoName := mux.Vars(r)["repoName"]
+	results, err := getResults(host, orgName, repoName)
+
+	if err == errorVerifyingFilepath {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		log.Println("error verifying filepath:", err)
```

Comment:
You can make this more generic by simply writing:
```
if err != nil{
  http.Error(w, err.Error(), http.StatusInternalServerError)
  log.Println("err: %v", err)
}
```

`err` already contains the information about the error.
This way you take care of *all* errors with at once